spec/request/folders_request_spec.rb

```diff
@@ -65,4 +65,34 @@
       end
     end
   end
+
+  describe '#destroy' do
+    context 'with valid folder id' do
+      let(:id) { 613_557 }
+
+      it 'response to DELETE #destroy' do
+        VCR.use_cassette("sm/folders/#{user_id}/delete_valid") do
+          delete "/v0/messaging/health/folders/#{id}"
+        end
+
+        expect(response).to be_success
+        expect(response).to have_http_status(:no_content)
+      end
+    end
+
+    context 'with non-existing id' do
```

Comment:
yea,  that's what i thought at first, but the api returned non that response when i tested it. The point here is really that it's returning whatever MHV said